crates/astria-conductor/src/cli.rs

```diff
@@ -8,6 +8,7 @@ pub struct Cli {
     #[serde(skip_serializing_if = "::std::option::Option::is_none")]
     pub celestia_node_url: Option<String>,
 
+    /// URL of the Tendermint node (sequencer/metro).
```

Comment:
```suggestion
    /// URL of the Tendermint node
```

this is the cometbft url